recipes/ndscope/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "ndscope" %}
+{% set version = "0.7.6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://git.ligo.org/cds/ndscope.git
+  git_tag: {{ version }}
```

Comment:
Please do not use the `git_url` and use a tarball instead: http://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos

Reply:
@scopatz, AFAICT, this package can _only_ be built from a clone of the repo, since it requires `setuptools_scm` to parse the version number from the tag. Maybe we can work to use a tarball for the next release.

Reply:
`setuptools_scm` should normally include the release version in the `sdist`. You don't plan to upload this to PyPI or somewhere else?

Reply:
You could upload an sdist tarball to GitHub releases, too if you need, and don't want to upload to PyPI. 

`git_url` is not strictly forbidden, but it does break the autoticker bot when it is used. When using `setuptools_scm` or `versioneer` or other tools, it is even more important to have a tarball, so that the version numbers makes sense. (conda will fail to install/activate packages that use versioneer when it isn't exactly a release number because the version number is "ver+hash", and the `+` character isn't valid). 

I don't merge PRs that use `git_url` because of all of the problems.